amundsen_application/static/js/components/SearchPage/index.tsx

```diff
@@ -91,6 +92,18 @@ export class SearchPage extends React.Component<SearchPageProps, SearchPageState
     }
```

Comment:
nit: This isn't part of your PR, but on another look I don't think we need to call `this.updatePageUrl()` in this method unless `selectedTab != currentTab` or `pageIndex != index`. Otherwise we're just updating the url with the exact parameters we just got from it. 

No need to make this change in this PR, but I wanted to call it out and we can clean that up on another PR.

Reply:
fixed this updatePageUrl call to only fix the url when selectedTab or pageIndex were out of sync with the state of the page